### PR TITLE
Instert configuration values for password policy

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Some configuration values were missing in sql. Refer to https://github.com/PrestaShop/PrestaShop/issues/29395
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | https://github.com/PrestaShop/PrestaShop/issues/29395
| How to test?      | refer to https://github.com/PrestaShop/PrestaShop/issues/29395
| Possible impacts? |

### CODE REVIEW

File: upgrade/sql/8.0.0.sql [hunk 1]
```diff
@@ -91,3 +91,10 @@ UPDATE `PREFIX_tab` SET wording = 'New & Experimental Features' WHERE `class_nam
 /* PHP:ps_update_tab_lang('Admin.Navigation.Menu', 'AdminFeatureFlag'); */;
 
 UPDATE `PREFIX_quick_access` SET `link` = 'index.php/sell/orders' WHERE `link` = 'index.php?controller=AdminOrders';
+
+/* Insert new password policy configuration values */
+INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES
+  ('PS_SECURITY_PASSWORD_POLICY_MAXIMUM_LENGTH', '72', NOW(), NOW()),
+  ('PS_SECURITY_PASSWORD_POLICY_MINIMUM_LENGTH', '8', NOW(), NOW()),
+  ('PS_SECURITY_PASSWORD_POLICY_MINIMUM_SCORE', '3', NOW(), NOW())
```
Comment (zuk3975) [inline upgrade/sql/8.0.0.sql:99]: related config names defined in https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Security/PasswordPolicyConfiguration.php#L56-L58

**Review by matthieu-rolland — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]
Thanks @zuk3975

**Review by sLorenzini — APPROVED** [review]
hello @zuk3975,

Thank you for this PR.
Tested an upgrade from 1.7.87 to 8.0.0 with PHP 7.3.

Upgrade successful and employees tab + add new employee is available and without errors:

https://user-images.githubusercontent.com/2854834/187642112-50381a68-2317-4786-bf11-d7bc34824a4f.mp4

QA approved ✅ 

Thank you

**Comment by matks** [comment]
Thank you @zuk3975 and @sLorenzini

**Pull request merged**